metabase/metabase
# Disable parameter mapping with native models

closes https://github.com/metabase/metabase/issues/44288
Demo https://www.loom.com/share/2718b1e2389046858343fa88a12a3b7c

Changes:
- Disallows parameter mapping to native models

How to verify:
- add a SQL (`SELECT * FROM ORDERS`) model to a dashboard and try to map it to a numeric parameter
- you should see a new error message

<img width="342" alt="Screenshot 2024-06-18 at 15 06 41" src="https://github.com/metabase/metabase/assets/8542534/9caa2171-6e54-4717-aaa8-f87e19bfd37a">
